.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+      compiler: clang
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
```

Comment:
Which version of clang do you need?

I think you should explicitly specify the version in packages field and matrix_eval environment, or you'll just use the default clang version 7. If ver.7 is what you want, then you don't need to specify apt-sources and matrix_eval at all.
https://docs.travis-ci.com/user/languages/c/#clang-on-linux

Current ci runs with clang 7.0.0. https://travis-ci.org/google/libsxg

Reply:
Or you might rather want to specify the version even if want to want is clang 7.0, to avoid build breakages from ubuntu upgrade.

Reply:
removed.